src/libraries/System.IO.Pipelines/tests/PipeReaderStreamTests.cs

```diff
@@ -38,7 +38,9 @@ public async Task DisposingPipeReaderStreamCompletesPipeReader(bool dataInPipe)
             for (int i = 0; i < 2; i++)
             {
                 s.Dispose();
+#if NETCOREAPP
```

Comment:
Moved this test file and added 2 minor changes to make it compile for netfx. So now we get new test coverage for PipeReaderStream on netfx.

Reply:
nit: most of them were switched to `#if NET` in runtime repo.